src/graphs.jl

```diff
@@ -365,13 +371,35 @@ end
                     _3d && push!(zseg, z[si], z[di], NaN)
                     push!(l_wg, wi)
                 end
+            if typeof(edgelabel) == Dict
+                if edge_label_exsists(edgelabel, (si, di))
+                    if dim == 2
+                        push!(edge_label_array,
+                              (0.25xsi + 0.75xdi, 0.25ysi + 0.75ydi, string(edgelabel[(si, di)])))
```

Comment:
I don't know this code super well, but shouldn't this be `xpts` and `ypts` so the labels follow along the curve?

Reply:
Yes, that will make things look a little nicer.